### ISSUE TITLE
Normal Subscription Link don't work in v2.2.7

### ISSUE THREAD

**Issue by rickyivan** [MHSanaei/3x-ui#2184]
**### Welcome

- [X] Yes, I'm using the latest major release. Only such installations are supported.
- [X] Yes, I'm using the supported system. Only such systems are supported.
- [X] Yes, I have read all WIKI document,nothing can help me in my problem.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, log, etc).

### Description of the problem,screencshot would be good

After I upgraded to latest version 2.2.7, subscription link failed to work, it returns "HTTP ERROR 500"
after I try to open link in browser it says:
This page isn’t working
---domain----  is currently unable to handle this request.
HTTP ERROR 500
---------
after I try to update sub on V2rayN it says:
invalid sub content
---------
after downgrading to version 2.2.6 it works again.
Tested on 2 different servers on different Providers and different domains. result is same. 2.2.7 normal sub fails to load content
Thanks
![Untitled](https://github.com/MHSanaei/3x-ui/assets/65889742/8c416244-8fb1-4d94-9280-6e91ff295cd9)

### Version of 3x-ui 2.2.7

<details>

```console
# Paste here
```

</details>

### x-ui log reports or xray log

<details>

```console
# paste log here
```

</details>**

**Comment by MHSanaei** [comment]
Edit that inbound and update without any changes, if not fixed share your config link without domain

**Comment by TeleDark** [comment]
> Edit that inbound and update without any changes, if not fixed share your config link without domain

This method worked

**Comment by rickyivan** [comment]
> Edit that inbound and update without any changes, if not fixed share your config link without domain

I upgraded to v2.2.7 again, then I tested 2 types of users and results:
1- Multi-Inbound user subs didn't work (subscription ID used in multiple inbounds for same user)
2- Single-Inbound user subs work fine
3- I added another inbound user config with same sub ID to previous single-inbound user, it's sub link worked fine (sub containing two configs)
4-I added 3rd config to previous "2 config Sub ID", it failed again
5-I updated these 3 inbound without any change, sub step 4 worked again!
6-I updated all inbounds, problem solved for all users! 
It seems subscription links containing more than 2 inbounds fail to work. but updating inbounds without any change, fixes problem.
Thanks for response
